src/ResourceManager/Cdn/Commands.Cdn/Endpoint/RemoveAzureCdnEndnpoint.cs

```diff
@@ -0,0 +1,89 @@
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Management.Automation;
+using System.Net;
+using Microsoft.Azure.Commands.Cdn.Common;
+using Microsoft.Azure.Commands.Cdn.Models.Endpoint;
+using Microsoft.Azure.Commands.Cdn.Properties;
+using Microsoft.Azure.Management.Cdn;
+using Microsoft.Azure.Management.Cdn.Models;
+
+namespace Microsoft.Azure.Commands.Cdn.Endpoint
+{
+    [Cmdlet(VerbsCommon.Remove, "AzureCdnEndpoint", ConfirmImpact = ConfirmImpact.High), OutputType(typeof(bool))]
+    public class RemoveAzureCdnEndpoint : AzureCdnCmdletBase
+    {
+        [Parameter(Mandatory = true, ParameterSetName = FieldsParameterSet, HelpMessage = "Azure Cdn endpoint name.")]
+        [ValidateNotNullOrEmpty]
+        public string EndpointName { get; set; }
+
+        [Parameter(Mandatory = true, ParameterSetName = FieldsParameterSet, HelpMessage = "Azure Cdn profile name.")]
+        [ValidateNotNullOrEmpty]
+        public string ProfileName { get; set; }
+
+        [Parameter(Mandatory = true, ParameterSetName = FieldsParameterSet, HelpMessage = "The resource group of the Azure Cdn Profile")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(Mandatory = true, ParameterSetName = ObjectParameterSet, ValueFromPipeline = true, HelpMessage = "The endpoint.")]
+        [ValidateNotNull]
+        public PSEndpoint CdnEndpoint { get; set; }
+
+        [Parameter(Mandatory = false, HelpMessage = "Do not ask for confirmation.")]
+        public SwitchParameter Force { get; set; }
+
+        public override void ExecuteCmdlet()
+        {
+            if (ParameterSetName == ObjectParameterSet)
+            {
+                ResourceGroupName = CdnEndpoint.ResourceGroupName;
+                ProfileName = CdnEndpoint.ProfileName;
+                EndpointName = CdnEndpoint.Name;
+            }
+
+            try
+            {
+                CdnManagementClient.Endpoints.Get(EndpointName, ProfileName, ResourceGroupName);
+            }
+            catch (AggregateException exception)
+            {
+                var errorResponseException = exception.InnerException as ErrorResponseException;
+                if (errorResponseException == null)
+                {
+                    throw;
+                }
+
+                if (errorResponseException.Response.StatusCode.Equals(HttpStatusCode.NotFound))
+                {
+                    throw new PSArgumentException(string.Format(
+                        Resources.Error_DeleteNonExistingEndpoint,
+                        EndpointName,
+                        ProfileName,
+                        ResourceGroupName));
+                }
+            }
+
+            ConfirmAction(
```

Comment:
ok, I see lots of other command has both -Force and SupportsShouldProcess in the code. Should we just directly remove the -Force and only use ShouldProcess?
For example:
\src\ResourceManager\Sql\Commands.Sql\Database\Cmdlet\RemoveAzureSqlDatabase.cs
This command has both Force and ShouldProcess